spellcast.py

```diff
@@ -88,6 +97,19 @@ def __init__(self):
         Reads words from a file and initializes various attributes needed for
         managing the game board.
         """
+        self.words = set()
+        self.letter_values = {}
+        self.letter_multipliers = defaultdict(lambda: 1)
+        self.word_multipliers = defaultdict(lambda: 1)
+        self.board = []
+        self.row_count = 0
+        self.column_count = 0
+        self.total_count = Counter()
+        self.board_value = {}
+        self.word_values = []
+        self.calculate_value = None

```

Comment:
It looks like `calculate_value` is only used in the recalculate function so it probably doesn't need to be a attribute of WordBoard.

Reply:
Done.